### DISCUSSION TITLE
[Feature] See cities when zoom in

### DISCUSSION THREAD

**Discussion by eerison** [immich-app/immich#13658] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello guys!

The map feature is a really nice feature to pin the places that we visit. 

It would be nice if we could zoom in and see the name of cities that we visited.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
I'm not sure what you mean? Place names are displayed on the map

![image](https://github.com/user-attachments/assets/50cdd1d9-0373-4982-9c04-2af98509ea83)

**Comment by eerison** [comment]
Ahhh then it's happening on android app only 👀

![Screenshot_20241022_155905](https://github.com/user-attachments/assets/a16fc19d-43af-4bd3-88d9-30ba81c3adb6)

**Reply by bo0tzz** [reply]
The app works fine for me too. Are you on the latest version?

**Reply by eerison** [reply]
It is the last version that appears for me on playstore 👀
![Screenshot_20241022_173931](https://github.com/user-attachments/assets/96484b5f-57c4-430e-95cd-113bf807ead2)

**Reply by alextran1502** [reply]
This is the caching issue with the map. Unfortunately right now the only way to resolve it is to reinstall the app. Unless @zackpollard had other ideas

**Reply by eerison** [reply]
You are right! 

After I reinstall the app it is working as expected! 

As it's a already known issue,  I guess I can mark this discussion as solved.

Thank you guys for your support ❤️
